src/coreclr/nativeaot/Runtime/thread.cpp

```diff
@@ -1278,6 +1279,49 @@ FCIMPL0(Object**, RhGetThreadStaticStorage)
 }
 FCIMPLEND
 
+#ifdef TARGET_UNIX
+#define NEWTHREAD_RETURN_TYPE size_t
+#else
+#define NEWTHREAD_RETURN_TYPE uint32_t
+#endif
+
+EXTERN_C STDCALL NEWTHREAD_RETURN_TYPE RhThreadEntryPoint(void* pContext)
```

Comment:
```suggestion
static NEWTHREAD_RETURN_TYPE RhThreadEntryPoint(void* pContext)
```
Less ceremony? We prefer to omit explicit calling conventions if possible.